### ISSUE TITLE
Documentation request: improve documentation for Select Options

### ISSUE THREAD

**Issue by teppokoivula** [processwire/processwire-requests#465]
The docs page for Select Options doesn't currently seem to explain how one is supposed to check if a specific option is selected within an option field that allows multiple values. Perhaps it's just me, but this seems like a pretty common use case, so I feel it should be covered.

## What _didn't_ work

The docs include an example of iterating over selected values, but since that doesn't seem to make much sense in this case, here's what I initially tried — and for the record I use this fieldtype _very_ rarely, so sorry for the rookie mistake 😅

```php
if ($page->options_field == 'option_value') {
```

While that seemed to work at first, it resulted in weird behaviour later. I do realize now that this is covered in the docs ("when you access the title (or other) property from the field, it simply gives you the value for that property from the first item"), but admittedly I spent quite a while trying to figure out how I'd managed to misconfigure the field _before_ spotting this note in the docs.

(I'm only mentioning this because to me it seems like a pretty big gotcha: comparing the value stored in a multi-value Select Options field with singular comparison value may seem to work, while it actually doesn't, so the mistake is easy to miss.)

## Solution?

Long story short, this is what I ended up with — but since this is not documented, I'm not sure if it's what I'm supposed to do:

```php
if ($page->options_field->hasValue('option_value')) {
    // ...
}
```

Would be nice if the docs could be extended to cover this use case as well 🙂

**Comment by ryancramerdesign** [comment]
Thanks @teppokoivula that's a good idea, I have added a new section in the documentation here:
https://processwire.com/docs/fields/select-options-fieldtype/#checking-if-an-option-is-selected

**Comment by teppokoivula** [comment]
Very nice, thanks again 🙂
